api/include/opentelemetry/trace/semantic_conventions.h

```diff
@@ -1,6 +1,12 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
+// NOTE:
```

Comment:
Can you also add a note in the "versioning" document that denotes that headers considered unstable will be documented as such?

Reply:
This is done now.